Transports/com.mlapi.contrib.transport.photon-realtime/README.md

```diff
@@ -10,23 +10,32 @@ Follow the [transport installation guide](../README.md) to add the Photon Realti
 Once added, please follow the following steps
 
 1. Access Wizard from Window → Photon Realtime → Wizard
-2. Click next until you get to the Photon Cloud Setup Part
-3. If you don't already have an account or an App Id, (More information in the [License](/Transports/com.mlapi.contrib.transport.photon-realtime/Runtime/Photon/LICENSE)) click on the Visit Dashboard button in order to create one	
-    1.  If you just created an account  for the first time you need to also create a New Application from Photon's Dashboard
-    2. NOTE! The Photon Type for this transport is: `Photon Realtime`
 
-    ![Img/Photon-App.png](Img/Photon-App.png)
+2. Click next until you get to the Photon Cloud Setup Part
 
-4. Once an Application is setup, you will be able to get it's App ID from Photon's Dashboard, copy that to your clipboard
-    - NOTE! This App Id **must** be shared with other people in order to be able to connect to your Photon Room!
+3. To complete the setup, you need a "Photon Realtime" AppId. If you know yours, enter it and click the Setup button.
+    Alternatively, enter your email address and click Setup. If possible, a new Photon account gets created (more information in the [License](/Transports/com.mlapi.contrib.transport.photon-realtime/Runtime/Photon/LICENSE)) and the AppId is fetched. This completes the setup.
+    If you already got a Photon account, the Wizard can't fetch the AppId, so click the "Visit Dashboard" button and login to the Dashboard to access your AppIds.
 
-    ![Img/Photon-Dashboard.png](Img/Photon-Dashboard.png)
+    1. If you just created an account  for the first time you need to also create a New Application from Photon's Dashboard
 
-5. Go back to the Photon Wizard Window of your Unity Project and paste the App Id there
-6. Click on Setup
-    - It should show something like: "Your AppId is now applied to this Project **Done** "
-7. Click on Next, then Done, you should be all setup now!
-8. You can safely quit the Photon Wizard now!
+    2. NOTE! The Photon Type for this transport is: `Photon Realtime`
+       ![Img/Photon-App.png](Img/Photon-App.png)
+    
+    3. Once an Application is setup, you will be able to get it's AppID from Photon's Dashboard, copy that to your clipboard
+       ![Img/Photon-Dashboard.png](Img/Photon-Dashboard.png)
+    
+       NOTE! To connect others to your Photon Room, the AppId **must be the same** in all clients!
```

Comment:
@Cosmin-B I was wondering what are those NOTE! things for? Do we need to reformat them for markdown, are they like block quotes or?